rasa/utils/tensorflow/temp_keras_modules.py

```diff
@@ -225,35 +444,35 @@ def fit(
                 if self.stop_training:
                     break
 
-            # If _eval_data_handler exists, delete it after all epochs are done.
+            # If eval data_hanlder exists, delete it after all epochs are done.
```

Comment:
```suggestion
            # If eval_data_handler exists, delete it after all epochs are done.
```
actually this typo is also in the original TF code.. so maybe no point in fixing here 🤷‍♀️